atk/README.md

```diff
@@ -1,12 +1,41 @@
 # atk 
 [Project site](https://gtk-rs.org/) | [Online documentation](https://gtk-rs.org/docs-src/)
 
-__Rust__ bindings and wrappers for __Atk__.
+__Rust__ bindings and wrappers for __Atk__, part of [gtk-rs](https://github.com/gtk-rs/gtk-rs).
 
 ## Minimum supported Rust version
 
 Currently, the minimum supported Rust version is `1.48.0`.
 
+## Documentation
+
+ * [Rust API - Stable](https://gtk-rs.org/docs/atk/)
```

Comment:
we do have devel version of the docs as well, maybe it's worth mentioning it here ? especially that building the docs locally require knowing about the `dox,embed-lgpl-docs` trick

Reply:
We do for gtk-rs too? :o Where is it?

Reply:
https://gtk-rs.org/gtk-rs/atk/, they are built by the CI and published at gh-pages

Reply:
Actually, we should probably move them to use something similar to the url format of what we have in gtk4-rs with gtk-rs.org/gtk4-rs/git/docs & we can do the stable/docs as well if you want.

Reply:
Yes we should definitely do that :) Ideally before merging this so that it can be correct from the beginning.

Reply:
I opened a PR for the git docs https://github.com/gtk-rs/gtk-rs/pull/438

Reply:
The git docs are available at https://gtk-rs.org/gtk-rs/git/docs/